zeebe/broker/src/test/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirectorTest.java

```diff
@@ -664,6 +664,20 @@ public void shouldRecoverFromStartWithNonUpdatingExporter() throws Exception {
     assertThat(exporters.get(1).getExportedRecords()).isEmpty();
   }
 
+  @Test
+  public void shouldStartContainersSoftPaused() {
```

Comment:
:x: This test is not enough. This is testing the implementation detail, not the actual expected behavior. I would suggest to verify the behavior. That is - new events are exported, but the position is not updated.